### DISCUSSION TITLE
How do I find out who replied to a tweet?

### DISCUSSION THREAD

**Discussion by wskxjtzq** [tweepy/tweepy#1865] [Questions] [answered]
How to find out who replied to a tweet? I need to know which users replied to a certain tweet. How to do it?

**Comment by wskxjtzq** [comment]
I am using V2. I didn't find any interface that can accomplish this task. Please help.

**Comment by Harmon758** [comment, marked as answer]
You can search by [conversation ID](https://developer.twitter.com/en/docs/twitter-api/conversation-id).
